battleship-client/src/main/java/com/academy/solid/nie/client/communication/IpValidator.java

```diff
@@ -12,21 +12,28 @@
 
     private final String digitsOnly = "[0-9]\\d*";
```

Comment:
static

Reply:
I told you about that - it shouldn't be static. Only tools wants it to be